pkg/virt-controller/services/template.go

```diff
@@ -65,7 +64,7 @@ func (t *templateService) RenderLaunchManifest(vm *v1.VM) (*kubev1.Pod, error) {
 		VolumeMounts: []kubev1.VolumeMount{
 			{
 				Name:      "sockets",
-				MountPath: socketDir,
+				MountPath: t.socketBaseDir,
```

Comment:
@rmohr 
I don't feel good about this change.  Previously each virt-launcher pod was isolated to it's own directory. With this change, all sock files are in a single common base directory that every virt-launcher pod has access to. 

Can we go back to the way this was?

Reply:
Done, However, we now leave empy directories behind. I can't delete them reliably while there might be a chance that the virt-launcher pod is still running. Any ideas?

Reply:
How concerned are you about this in general? At the end, we will have to start the virt-launcher pod via a privileged user to allow hostMounts, we control the image registry, and we can enter the virt-launcher namespace in a subsequent patch to also check if the virt-launcher contains a shared k8s secret between virt-launcher and virt-handler.

Reply:
I created an issue #421 for us to follow up on this.  I don't want to block this patch for that discussion.

In general, I'd like to restrict access for each component to only what it needs to see. That limits the possibilities to some degree.